packages/extension-ui/.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     'react/jsx-newline': ['warn', { prevent: true, allowMultilines: false }],
     'sort-keys': 'off',
     'react/jsx-no-bind': 'off',
+    '@typescript-eslint/no-floating-promises': 'off'
```

Comment:
This rule prevents applying some _common_ control flows.

Reply:
on second thought, perhaps we should reconsider :sweat_smile: